src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -134,6 +134,7 @@ internal static class IDEDiagnosticIds
         public const string LegacyFormatSuppressMessageAttributeDiagnosticId = "IDE0077";
 
         public const string RemoveConfusingSuppressionForIsExpressionDiagnosticId = "IDE0080";
+        public const string ParenthesesStyleDiagnosticId = "IDE0082";
```

Comment:
probably too vague.
also, we need an option for methods and an option for properties.

Reply:
>  also, we need an option for methods and an option for properties.

These should be different IDs? e.g: 82 for methods and 83 for properties?

Reply:
yes.  i would expect to get different ids here.  and i would expect to be able to fix-all each group of issues independently.

honestly, i would just do these as two full different analyzers/fixers.

Reply:
> honestly, i would just do these as two full different analyzers/fixers.

Great. I'll continue soon.